src/native/libs/System.Native/pal_console_wasi.c

```diff
@@ -117,3 +117,6 @@ int32_t SystemNative_InitializeTerminalAndSignalHandling(void)
 {
     return true;
 }
+
+void SystemNative_UninitializeTerminalAndSignalHandling(void)
```

Comment:
This should fix the build.

```suggestion
void SystemNative_UninitializeTerminal(void)
```